docs/AboutUs.md

```diff
@@ -6,18 +6,21 @@ title: About Us
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
+## Mentors
+* Lecturer : Prof Damith Chatura Rajapakse
+* Supervisor : Choo Qi Le Aaron
 
 ## Project team
 
 ### Oh Jun Ming
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ohjunming.png" width="200px">
 
 [[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/OhJunMing)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Project Advisor
+* Role: Documentation User Guide
```

Comment:
Hey can you remove the role part, since for now I think we are randomly distributing tasks so that we all get to learn equally? We discussed this yesterday i guess. otherwise LGTM!

Reply:
Hey,
yup yup, my mistake
Already amended it.
Can check it again:)